common/cuda_hip/base/dev_lapack_bindings.hpp

```diff
@@ -0,0 +1,18 @@
+// SPDX-FileCopyrightText: 2017 - 2025 The Ginkgo authors
```

Comment:
I thought the auto generation should only have 2025?

Reply:
Oh, I know what happened, I copied the framework from common/cuda_hip/base/cusparse_bindings.hpp (or other relevant files for other bindings) to start and left the copyright line in, so that's why it wasn't correct even after auto generation. I will fix it.

Reply:
This should be fixed in 688b4c4a539cdcb573961a0d0351bdbc3d73e2d7.